generators/app/templates/src/app/screens/Registration/index.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useCallback } from 'react';
+import i18next from 'i18next';
+
+import FormInput from '~components/FormInput';
+import PATHS from '~components/Routes/paths';
+
+import { FIELDS } from './constants';
+import styles from './styles.module.scss';
+
+function RegistrationContainer() {
+  /* eslint-disable @typescript-eslint/no-unused-vars */
+  const handleSubmit = useCallback((event: React.FormEvent<Element>) => {
+    // TODO: Implement
+  }, []);
+
+  const handleInputChange = useCallback((event: React.FormEvent<Element>) => {
+    // TODO: Implement method
+  }, []);
+
+  return (
+    <form className={`column center full-width m-top-8 ${styles.formContainer}`} onSubmit={handleSubmit}>
+      <div className="column center m-bottom-3">
+        <h1 className="m-bottom-1">{i18next.t('Registration:registration')}</h1>
+        <h2>{i18next.t('Registration:registrationExplanation')}</h2>
+      </div>
+      <div className={`row space-between m-bottom-2 full-width ${styles.sectionContainer}`}>
+        <FormInput
+          className="full-width"
+          label={i18next.t('Registration:name')}
+          name={FIELDS.name}
+          inputType="text"
+          inputClassName={`m-right-1 m-bottom-2 full-width ${styles.input}`}
+          placeholder={i18next.t('Registration:namePlaceholder') as string}
+          onChange={handleInputChange}
+        />
+        <FormInput
+          className="full-width"
+          label={i18next.t('Registration:surname')}
+          name={FIELDS.surname}
+          inputType="text"
+          inputClassName={`m-left-1 m-bottom-2 full-width ${styles.input}`}
```

Comment:
prefer using m-right-1 in the labelClassName. We try to avoid margin left and top